docs/spec/historical.rst

```diff
@@ -188,25 +188,30 @@ When checking Python 2.7 code, type checkers should treat the ``int`` and
 type ``str`` as well as ``unicode`` should be acceptable.
 
 
-Positional-only arguments
--------------------------
+Positional-only parameters
+--------------------------
 
 Some functions are designed to take their arguments only positionally,
 and expect their callers never to use the argument's name to provide
 that argument by keyword. Before Python 3.8 (:pep:`570`), Python did
-not provide a way to declare positional-only arguments.
+not provide a way to declare positional-only parameters.
+
+To support positional-only parameters in code that must remain compatible
+with older versions of Python, type checkers should support the following
+special case: all parameters with names beginning with ``__`` are assumed to 
+be positional-only unless their names also end with ``__``::
```

Comment:
Nit: The phrasing here seems to draw attention to the trailing `__`. Suggestion:
```suggestion
special case: all parameters with names that begin but don't end with ``__``
are assumed to be positional-only::
```

Reply:
I didn't modify this text from the earlier version, but I agree that your new phrasing is easier to understand. I'm OK with updating it for clarity.